File: java/src/main/java/com/iteso/nintendo/attacks/Hex.java
package com.iteso.nintendo.attacks;

public class Hex implements Attack {
    static int damage;
    private static final String name = "Hex";

    @Override
    public String attackName() {
        return name;
    }

    @Override
    public int attackDamage() {
        return this.damage;
    }

    @Override
    public void setDamage(int damage) {
        this.damage = damage;
    }
}

File: python/Hex.py
class Hex:
    def __init__(self):
        self.damage = None
        self.name = 'Hex'

    def attackName(self):
        return self.name

    def attackDamage(self):
        return self.damage

    def setDamage(self, damage):
        self.damage = damage
